usdigitalresponse/usdr-gost
# Update Node from v16 to v20.

## Description

I suggest we upgrade to ~~Node 18~~ Node 20. Node 16 [reached end-of-life](https://nodejs.org/en/blog/announcements/nodejs16-eol) on September 11th, 2023. ~~Node 18 is in maintenance mode and will be until April 2025.~~ Node 20 is the currently active version of Node and will be supported through mid-2026.

## Screenshots / Demo Video
No user-facing changes.

## Testing

1. Execute `nvm use` to switch to the new version of Node.
2. Run `yarn install` and check for any errors.
3. Run `yarn test` and check for errors.
4. Start up the app with rebuilt images using `docker compose up -d --build`. Click around and check that everything works.
5. Execute `yarn cypress:run` to check that end-to-end tests still work.

### Automated and Unit Tests

### Manual tests for Reviewer

## Checklist
